pages/common/git-diff.md

```diff
@@ -14,6 +14,10 @@
 
 `git diff --staged`
 
+- Show changes from all commits since a given date/time:
+
+`git diff 'HEAD@{3 days ago}'`
```

Comment:
What other values are supported ? For eg. hours/months/years. If possible, it would be good to put all the values separated by |